products/kuma.md

```diff
@@ -0,0 +1,51 @@
+---
+title: kuma
+category: app
+tags: cncf configuration-management
+permalink: /kuma
+alternate_urls:
+-   /kuma
+releasePolicyLink: https://github.com/kumahq/kuma/blob/master/versions.yml
+changelogTemplate: https://github.com/kumahq/kuma/releases/tag/__LATEST__
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+  methods:
```

Comment:
Note: it's possible to develop a script in https://github.com/endoflife-date/release-data to automatically get the dates from https://github.com/kumahq/kuma/blob/master/versions.yml.

Reply:
script here
https://github.com/endoflife-date/release-data/pull/359